eventeditor/__main__.py

```diff
@@ -180,6 +186,7 @@ def closeEvent(self, event) -> None:
     def readSettings(self) -> None:
         settings = qc.QSettings()
         ai.set_rom_path(settings.value('paths/rom_root'))
+        aj.set_actor_json_path(settings.value('paths/actor_json_root'))
```

Comment:
nit: I would recommend calling this actor_definitions as just "actor" is a bit vague. JSON shouldn't be in the name as we don't want to be tied to a specific storage format.